modules/core/src/main/java/org/apache/ignite/mxbean/CacheMetricsMXBean.java

```diff
@@ -151,10 +151,18 @@
     @MXBeanDescription("Number of non-null values in the cache.")
     public int getSize();
 
+    /** {@inheritDoc} */
+    @MXBeanDescription("Number of non-null values in the cache.")
+    public long getSizeLong();
+
     /** {@inheritDoc} */
     @MXBeanDescription("Number of keys in the cache (possibly with null values).")
     public int getKeySize();
 
+    /** {@inheritDoc} */
+    @MXBeanDescription("Number of keys in the cache (possibly with null values).")
```

Comment:
maybe to add to the description "as a long type"?

Reply:
Done.